cli/ballerina-packerina/src/main/java/org/ballerinalang/packerina/cmd/RunCommand.java

```diff
@@ -115,25 +116,26 @@ public void execute() {
                     "no ballerina program given.",
                     "ballerina run {<bal-file> | <module-name> | <executable-jar>}",
                     true);
-    
+
             Runtime.getRuntime().exit(1);
             return;
         }
-    
-        // enable remote debugging
+
+        // enable remote debugging.
         if (null != this.debugPort) {
```

Comment:
Shall we pass debug port directly to RunExecutableTask then we do not need to set any system property. Also in RunExecutableTask we can check the null != this.debugPort add java agent args.

Reply:
Packerina implementation has used the same implementation(system property) for all the other commands (push,pull,search, etc) as well. WDYT?

Reply:
+1 use system property.